tests/pytests/test_vibanalysis.py

```diff
@@ -947,13 +998,33 @@ def test_harmonic_analysis_vs_cfour(subject, request):
         pytest.param('nh3'),
         pytest.param('h2co', marks=pytest.mark.quick),
         pytest.param('hooh'),
+        pytest.param('ne'),
     ],
-    ids=['CO2', 'ethene', 'methane', 'ammonia', 'formaldehyde', 'HOOH_TS'])
-def test_hessian_vs_cfour(subject, dertype, request):
+    ids=['CO2', 'ethene', 'methane', 'ammonia', 'formaldehyde', 'HOOH_TS', 'atom'])
+@pytest.mark.parametrize(
+    "scf_type", [
+        "pk",
```

Comment:
So we want to compare both, `df` and `pk` results now? Is it just making sure it's easier to check where problems are (ie if `pk` matches `pk`, it's a bug in density fitting, otherwise it's in the hessian itself?

Reply:
This file is the compare-to-other-qc-program test of freq results so all the refs are independent ( and non-FD). The new df check will make sure that df isn’t too far off from “right” in the quantities user care about like omega and IR intensity. I’m going to propose that every new analytic Hess ( like uhf) add to this file